src/rust/ide/view/graph-editor/src/component/node.rs

```diff
@@ -649,6 +645,7 @@ impl Node {
             layer_on_not_hover <- preview_visible.on_true().map(|_| visualization::Layer::Front);
             layer              <- any(layer_on_hover,layer_on_not_hover);
             model.visualization.frp.set_layer <+ layer;
+            eval layer ((l) model.error_visualization.frp.set_layer.emit(l));
```

Comment:
What this line has to do with "showing node output type under the node" (which is the subject of this PR)?